modules/rosa-delete-objects.adoc

```diff
@@ -109,6 +109,55 @@ Delete a cluster named `mycluster`.
 $ rosa delete cluster --cluster=mycluster
 ----
 
+[id="rosa-delete-external-auth-provider_{context}"]
+== delete external-auth-provider
+
+Deletes an external authentication provider from a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete external-auth-provider <name_of_external_auth_provider> --cluster=<cluster_name> | <cluster_id> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|Required: The name or ID (string) of the cluster from which the external auth provider will be deleted.
```

Comment:
```suggestion
|Required. The name or ID string of the cluster the external auth provider will be deleted from.
```
Putting a period after "required" just to be consistent with the other tables. Also, see other comment about parenthesis.